src/Pool.sol

```diff
@@ -240,13 +240,9 @@ abstract contract Pool {
         }
 
         // Dripped when collected
-        Sender storage sender = senders[receiverAddr];
-        if (collected > 0 && currReceivers.length > 0 && sender.dripsFraction > 0) {
-            uint256 drippable = (uint256(collected) * sender.dripsFraction) / MAX_DRIPS_FRACTION;
-            uint128 totalAmtPerSec = _totalAmtPerSec(currReceivers);
-            for (uint256 i = 0; i < currReceivers.length; i++) {
-                dripped += uint128((drippable * currReceivers[i].amtPerSec) / totalAmtPerSec);
-            }
+        if (collected > 0 && currReceivers.length > 0) {
+            Sender storage sender = senders[receiverAddr];
+            dripped = uint128((uint160(collected) * sender.dripsFraction) / MAX_DRIPS_FRACTION);
```

Comment:
👍  True, the iteration is not required because we don't want to modify the receivers in the collectable